tests/compat.sh

```diff
@@ -887,8 +887,9 @@ if type lsof >/dev/null 2>/dev/null; then
         done
     }
 else
+    echo "Warning: lsof not available, wait_server_start = sleep 1"
```

Comment:
Please change this to `… = sleep 2` or to `… = sleep` (displaying the timeout value isn't important, but if it's displayed, it should be correct).